libs/barista-components/filter-field/src/filter-field.ts

```diff
@@ -494,6 +496,15 @@ export class DtFilterField<T = any>
   /** Whether the filter field or one of it's child elements is focused. */
   private _isFocused = false;
 
+  /** Whether to allow focusing the tags when navigating in the input element using arrow keys */
+  private allowArrowKeyFocusOnTags = false;
+
+  /** Whether the focus in the suffix tags or not */
+  private inSuffix = false;
```

Comment:
I'd reword the variables to e.g. _hasFocusInSuffix - also think about using one focus state variable instead as well. The focus can never be in both (suffix and prefix) so they are mutually exclusive. If you combine them into one variable, it gets easier to avoid impossible state. 
pseudo code:
```
private _currentFocusState: 'none' | 'input' | 'inputstart' | 'inputend' | 'prefix' | 'suffix' = 'none';
```